### ISSUE TITLE
Input range mismatch?

### ISSUE THREAD

**Issue by VicJones5678** [Cadene/pretrained-models.pytorch#188]
Regarding: pretrainedmodels/models/xception.py

Normalizing an image as below will result in a range -1..1 (not 0..1 as specified).
Please can you confirm if the input range should be -1..1 or 0..1?

Many thanks,
   -Vic.

"""...
normalize = transforms.Normalize(mean=[0.5, 0.5, 0.5],
                                  std=[0.5, 0.5, 0.5])
..."""

pretrained_settings = {
    'xception': {
        'imagenet': {
...
            'input_range': [0, 1],    <-- ???
            'mean': [0.5, 0.5, 0.5],
            'std': [0.5, 0.5, 0.5],
...
